### PR TITLE
[M482] Fix missing bleaching download

### PR DESCRIPTION
**Pull request by tienhoah**
**Steps to test**:

- Find a project to look for bleaching in download modal.
- **Example**:
   -  Filter sites by project `2023 Coral Bleaching_CoralAtlas`.
   -  Select any site marker.
   -  Click download.
   -  Bleaching download should appear if it's in the project.
- **Screenshot**:
Screenshot 2023-05-26 at 2 03 20 PM

### CODE REVIEW

**Comment by tienhoah** [comment]
[include bleaching in download list](https://trello.com/c/kP6YXc5M/482-include-bleaching-in-download-list)

**Comment by tienhoah** [comment]
[include bleaching in download list](https://trello.com/c/kP6YXc5M/482-include-bleaching-in-download-list)

File: src/components/DownloadDataModal.js [hunk 1]
```diff
@@ -44,8 +44,13 @@ const DownloadDataModal = ({ currentSelectedSite, sites }) => {
...
   const availableProtocols = useMemo(() => {
     const allSites = sites.map(site => site[1]).flat()
-    const allProtocols = allSites.map(site => Object.keys(site.protocols)[0])
-    const uniqueAvailableProtocol = [...new Set(allProtocols)]
+    const allProtocols = allSites.map(site => Object.keys(site.protocols)).flat()
+
+    // Both keys 'colonies_bleached' and 'quadrat_benthic_percent' are part of Bleaching protocol.
+    // this filter is based on the setup in transect-protocol.js as 'quadrat_benthic_percent' is used to determined as Bleaching.
+    const uniqueAvailableProtocol = [...new Set(allProtocols)].filter(
+      protocols => protocols !== 'colonies_bleached',
```
Comment (julia-rom) [inline src/components/DownloadDataModal.js:52]: would it work to use `protocols === 'quadrat_benthic_percent'` here to be more explicit?

**Review by julia-rom — COMMENTED** [review]
code clarification comment below

File: src/components/DownloadDataModal.js (hunk 1 above)
Comment (tienhoah) [reply src/components/DownloadDataModal.js:52]: Here are list of protocols, both 'colonies_bleached' and 'quadrat_benthic_percent' do mean the same thing here, but I have to pick one to render this unique list. 

Screenshot 2023-06-01 at 12 09 00 PM

Comment (julia-rom) [reply src/components/DownloadDataModal.js:53]: ok so for `uniqueAvailableProtocol` we are filtering out bleached records then? was just looking at the code comments

Comment (tienhoah) [reply src/components/DownloadDataModal.js:53]: it still filter all the protocols including bleaching, `colonies_bleached` is an **extra** key, but it also refers to Bleaching (I could pick either `colonies_bleached` or `quadrat_benthic_percent`)
it is confusing though :)

Comment (julia-rom) [reply src/components/DownloadDataModal.js:53]: ok thanks for explaining - approved in that case!

**Pull request merged**